Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -236,9 +242,10 @@ void Update()
                 return;
             }
 
-            // NetworkTransform is interpolated - we can just apply it's position value to our visual object
-            transform.position = m_PhysicsWrapper.Transform.position;
-            transform.rotation = m_PhysicsWrapper.Transform.rotation;
+            transform.SetPositionAndRotation(m_PositionLerper.LerpPosition(transform.position,
```

Comment:
mmm thinking of it, shoudn't we lerp only for host? pure server and pure clients shouldn't need to do this, no? just host for visual smoothing when network transform doesn't do it?